app/models/manageiq/providers/storage_manager/cinder_manager.rb

```diff
@@ -3,33 +3,6 @@ class ManageIQ::Providers::StorageManager::CinderManager < ManageIQ::Providers::
 
   include ManageIQ::Providers::StorageManager::BlockMixin
 
-  # Auth and endpoints delegations, editing of this type of manager must be disabled
-  delegate :authentication_check,
-           :authentication_status,
-           :authentication_status_ok?,
-           :authentications,
-           :authentication_for_summary,
-           :cinder_service,
-           :connect,
-           :verify_credentials,
-           :with_provider_connection,
-           :address,
-           :ip_address,
-           :hostname,
-           :default_endpoint,
-           :endpoints,
-           :to        => :parent_manager,
-           :allow_nil => true
-
-  supports :cinder_service do
-    if parent_manager
-      unsupported_reason_add(:cinder_service, parent_manager.unsupported_reason(:cinder_service)) unless
-        parent_manager.supports_cinder_service?
-    else
-      unsupported_reason_add(:cinder_service, _('no parent_manager to ems'))
-    end
-  end
```

Comment:
This is about swift but cinder had the same issue, but the ems_refresh code for cinder was moved to openstack a long time ago.  I'm not sure if there is a benefit of having a CinderManager and SwiftManager in core at all, but if there is it shouldn't depend on having a parent_manager to work.